docs/source/markdown/podman-push.1.md.in

```diff
@@ -40,6 +44,9 @@ $ sudo podman push myimage docker-daemon:docker.io/library/myimage:33
 
 # Push to a tarball in the OCI format
 $ podman push myimage oci-archive:/tmp/myimage
+
+# Push to a directory in the OCI format with an explicitly specified tag
+$ podman push myimage oci:/tmp/myimage:tag
```

Comment:
This is a counterintuitive situation for many users. Please add a note making it unambiguously clear that the destination directory is `/tmp/myimage` and that `tag` is an annotation:
```
# Push to a directory ...
$ podman push myimage oci:/tmp/myimage:myname/mytag
$ jq '.manifests[].annotations' /tmp/myimage/index.json
{
  "org.opencontainers.image.ref.name": "myname/mytag"
}
```